src/javascript/app/pages/cashier/dp2p.js

```diff
@@ -61,13 +62,28 @@ const DP2P = (() => {
                 `;
                 el_main_css.rel = 'stylesheet';
 
+                const p2pSubscribe = (request, cb) => {
+                    SubscriptionManager.subscribe(Object.keys(request)[0], request, cb);
```

Comment:
can you add a comment about how request object should have the first param be the msg_type?

Reply:
sure